### ISSUE TITLE
Loading images as int8 degrade performance drastically...

### ISSUE THREAD

**Issue by hadikazemi** [fchollet/keras#4291]
When I load training images using keras im_load() function, the images are loaded as float32 (I guess) and my classifier gives about %86 accuracy. However, when I load the training data using opencv (it loads images as int8) the results of the new trained model degrade to %79. 
BTW, I normalized the images dividing by 255.0 before training.
```python
images = imgs /255.0 #imgs is float32 in the former case and int8 in the latter
```
What is the reason of this huge degradation?
